demo-client/src/main/java/com/illia/client/http_client/MyHttpClientImpl.java

```diff
@@ -0,0 +1,75 @@
+package com.illia.client.http_client;
+
+
+import com.illia.client.config.ClientConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import org.springframework.web.client.RestTemplate;
+
+import java.io.File;
+
+
+@Slf4j
+@Component
+public class MyHttpClientImpl implements MyHttpClient {
+
+    @Autowired
+    ClientConfig clientConfig;
+
+    @Override
+    public String uploadFile(String fileName, File file) {
+
+        var operation = "/uploadFile";
```

Comment:
define these variables smth like: 
private static final UPLOAD_FILE_PATH_PATTERN=/uploadFile?filename=%s"
private static final DOWNLOAD_FILE_PATH_PATTERN=/downloadFile?filename=%s"

you also can extract it to separate class containing only path variables, as we will have more of them in the near future